jcli/src/jcli_lib/rest/v0/message/mod.rs

```diff
@@ -57,14 +57,19 @@ fn get_logs(args: RestArgs, output_format: OutputFormat) -> Result<(), Error> {
 fn post_message(args: RestArgs, file: Option<PathBuf>) -> Result<(), Error> {
     let msg_hex = io::read_line(&file)?;
     let msg_bin = hex::decode(&msg_hex)?;
-    let _fragment =
+    let fragment =
         Fragment::deserialize(msg_bin.as_slice()).map_err(Error::InputFragmentMalformed)?;
+    let fragment_id = post_fragment(args, fragment)?;
+    println!("{}", fragment_id);
```

Comment:
This needs to be replaced with a more informative message.

Reply:
I kept it as it was. I understand that this may be used in order to pipe the output to something else. But I'm down to change it.